### PR TITLE
Italian translation

### PR DESCRIPTION
**Pull request by ltworf**

### CODE REVIEW

File: locale/xdecor.it.tf [hunk 1]
```diff
@@ -0,0 +1,154 @@
+# textdomain: xdecor
```
Comment (SmallJoker) [inline locale/xdecor.it.tf:1]: Wrong file name, should be `*.tr`.

File: locale/xdecor.it.tf [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+### hive.lua ###
+
+Artificial Hive=Favo artificiale
+Bees are busy making honey…=Le api sono occupate a fare il miele…
+Honey=Miele
+
+### itemframe.lua ###
+
+@1 (owned by @2)=@1 (proprietà di @2)
+Item Frame=Quadro
```
Comment (SmallJoker) [inline locale/xdecor.it.tf:74]: I don't think this translation matches perfectly. It's rather a "showcase"/"display case" than a painting.

**Review by SmallJoker — APPROVED** [review]
Seems good. Will merge in a few days unless there are objections.

**Comment by ltworf** [comment]
ok thanks

**Pull request merged**